File: C#Part1-Homework/03.OperatorsAndExpressions-Homework/08.PrimeNumberCheck/IsPrime.cs
namespace PrimeNumberCheck
{
    using System;

    class IsPrime
    {
        /* Write an expression that checks if given positive integer number n (n = 100) is prime 
           (i.e. it is divisible without remainder only to itself and 1).
         */
        static void Main()
        {
            Console.Write("Enter a number N = ");
            int numberN = int.Parse(Console.ReadLine());
            double root = Math.Sqrt(numberN);
            bool primeTest = true;
            for (int i = 2; i <= root; i++)
            {
                if (numberN % i == 0)
                {
                    primeTest = false;
                }
            }
            if (numberN == 1 || numberN <= 0)
            {
                primeTest = false;
            }
            Console.WriteLine("{0} is prime - {1}", numberN, primeTest);
        }
    }
}

File: JavaScript/Homework/Operators and Expressions - Homework/Problem 7 Is prime/IsPrime.js
// Write an expression that checks if given positive integer number n (n ≤ 100) is prime.

var num;

function isPrime(num) {
    if (isNaN(num) || !isFinite(num) || num % 1 || num < 2) {
        return false;
    }

    var m = Math.sqrt(num);
    for (var i = 2; i <= m; i++) {
        if (num % i == 0) {
            return false;
        }
    }
    return true;
}

console.log("Is 1 prime: " + isPrime(1));
console.log("Is 9 prime: " + isPrime(9));
console.log("Is 37 prime: " + isPrime(37));
console.log("Is 51 prime: " + isPrime(51));
console.log("Is -3 prime: " + isPrime(-3));
console.log("Is 0 prime: " + isPrime(0));
console.log("Is 97 prime: " + isPrime(97));